examples/api-routes-apollo-server-and-client/lib/init-apollo.js

```diff
@@ -0,0 +1,41 @@
+import ApolloClient from 'apollo-client'
+import { SchemaLink } from 'apollo-link-schema'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { HttpLink } from 'apollo-link-http'
+import { schema } from '../apollo/schema'
+
+let apolloClient = null
+
+function create (initialState) {
+  // Check out https://github.com/zeit/next.js/pull/4611 if you want to use the AWSAppSyncClient
+  const isBrowser = typeof window !== 'undefined'
+
+  const link = isBrowser
+    ? new HttpLink({
+      uri: '/api/graphql',
+      credentials: 'same-origin'
+    })
+    : new SchemaLink({ schema })
```

Comment:
while i think that's a super cool approach I wasn't aware of at all! :sparkles: 
I guess this will break as soon as you do/import sth strictly serverside in the resolver.

Try e.g. importing `import { Storage } from '@google-cloud/storage'` inside the resolver.
Perhaps you can work around this by only importing the schema on the server trough :thinking: